modules/ipc/include/hephaestus/ipc/zenoh/action_server/action_server.h

```diff
@@ -139,7 +139,7 @@ ActionServer<RequestT, StatusT, ReplyT>::ActionServer(SessionPtr session, TopicC
         session_, topic_config_, [this](const RequestT& request) { return onRequest(request); }))
   , request_consumer_([this](const RequestT& request) { return execute(request); }, std::nullopt) {
   request_consumer_.start();
-  LOG(INFO) << fmt::format("[ActionServer {}] Started Action Server.", topic_config_.name);
+  heph::log(heph::INFO, "started Action Server", "topic", topic_config_.name);
```

Comment:
while we are at it, let's move all the logs in heph library from INFO to DEBUG, in this way we allow the user to easily disable library log
```suggestion
  heph::log(heph::DEBUG, "started Action Server", "topic", topic_config_.name);
```